services/app-api/utils/formTemplates/formTemplateModifiers.ts

```diff
@@ -0,0 +1,69 @@
+import { AnyObject } from "../types";
+
+/*
+ * NOTE: This function is quite large, but it is temporary
+ * (until the release of this new MCPAR template in July 25th)
+ */
+export const handleTemplateForJulyMcparRelease = (
+  originalReportTemplate: any
+) => {
+  const reportTemplate = structuredClone(originalReportTemplate);
+  for (let route of reportTemplate.routes) {
+    // remove ILOS and Parity routes from template
+    if (
+      route.path === "/mcpar/program-information" ||
+      route.path === "/mcpar/plan-level-indicators" ||
+      route.path === "/mcpar/program-level-indicators"
+    ) {
+      route.children = removeIlosAndParityRoutes(route.children);
+    }
+    // remove Appeals and Grievances questions from template
+    if (route.path === "/mcpar/plan-level-indicators") {
+      route.children[3].children[0].drawerForm.fields =
+        removeAppealsAndGrievances(route.children);
+
+      // remove new radio button option from question D3.VIII.2 in template
+      route.children[5].modalForm.fields[1].props.choices.splice(6, 1);
+
+      // replace Program Integrity questions in template
+      route.children[6].drawerForm.fields = replaceProgramIntegrity(
+        route.children[6].drawerForm.fields
+      );
+
+      // remove new radio button option from question D1.X.6 in template
+      route.children[6].drawerForm.fields[5].props.choices.splice(1, 1);
```

Comment:
Would selecting the choice to remove by its content instead of index be a little safer? In case something gets moved?